crypto/x509v3/v3_addr.c

```diff
@@ -1260,7 +1277,11 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     /*
      * Trust anchor can't inherit.
      */
-    OPENSSL_assert(x != NULL);
+    if (!ossl_assert(x != NULL)) {
+        if (ctx != NULL)
```

Comment:
Position of this comment suggests that it's about ctx being NULL. This one is answered, i.e. ctx can be NULL. But question also is if the assertion is to be omitted or replaced. And it's apparently same question for all three occurrences, specifically if x being NULL is a *legitimate* option. Can one have non-zero length chain with NULLs it?